src/main/java/build/buildfarm/instance/server/NodeInstance.java

```diff
@@ -1405,7 +1405,7 @@ protected void logFailedStatus(Digest actionDigest, com.google.rpc.Status status
   }
 
   // this deserves a real async execute, but not now
-  @SuppressWarnings("ConstantConditions")
+  @SuppressWarnings({"ConstantConditions", "PMD.UseDiamondOperator"})
```

Comment:
What were the diamond suggestions from PMD?

Reply:
```
main/java/build/buildfarm/instance/server/NodeInstance.java:1479:	UseDiamondOperator:	Explicit type arguments can be replaced by a diamond: `new FutureCallback<>()`  
```